samples/WOCCatalog/WOCCatalog/AccelerateViewController.h

```diff
@@ -18,6 +18,30 @@
 #import <UIKit/UIKit.h>
 #import <Accelerate/Accelerate.h>
 
-@interface AccelerateViewController : UITableViewController
+@protocol PhotoSelectorControllerDelegate <NSObject>
+
+@required
+- (void)imageFromController:(UIImage *)image;
+@end
+
+@interface SelectorCell : UICollectionViewCell
+
+@property (nonatomic, strong) UIImageView* imageView;
+
+-(void)setImage:(UIImage *)image;
+
+@end
+
+@interface PhotoSelectorController : UICollectionViewController <UICollectionViewDataSource, UICollectionViewDelegate> {
+    NSString* place;
+    NSArray* images;
```

Comment:
These two ivars are private, so they can go into implementation instead of interface.  Also, place appears unused?